schemas/samples/B18/current.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$id": "/samples/B18/1.0.0",
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "title": "samples/B18",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "chemical_composition": { 
+      "title": "Chemical Composition",
+      "description": "Chemical composition of the sample, by percentage of chemical elements",
+      "type": "array",
+      "items": {
```

Comment:
As for the similar comment above, can we use a chemical formula instead here? Then there is a bit of magic required to go back to the %s and elements.